Generics.playground/Contents.swift

```diff
@@ -1,3 +1,55 @@
 import UIKit
 
-var str = "Hello, playground"
+struct CountedSet <Element: Hashable> {
+    
+    private(set) var dictionary = [Element: Int]()
+    
+    var count: Int? {
+        if dictionary.isEmpty {
+            return nil
+        } else {
+            return dictionary.count
+        }
+    }
```

Comment:
Probably just return an Int in this scenario, and have a separate isEmpty computed property